src/System.Windows.Forms/src/System/Windows/Forms/TaskDialog.cs

```diff
@@ -694,16 +449,49 @@ private static TaskDialogButton CreatePlaceholderButton(TaskDialogResult result)
         /// <returns>
         ///   The <see cref="TaskDialogButton"/> which was clicked by the user to close the dialog.
         /// </returns>
-        public unsafe TaskDialogButton ShowDialog(IntPtr hwndOwner)
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="hwndOwner"/> is <see langword="null"/>
+        ///   - or -
+        ///   <paramref name="page"/> is <see langword="null"/>.
+        /// </exception>
+        public static unsafe TaskDialogButton ShowDialog(IntPtr hwndOwner, TaskDialogPage page)
+        {
+            if (hwndOwner == null)
```

Comment:
This will always be `false` as it will result in comparing a non-null `IntPtr?` with a null `IntPtr?`.

Reply:
Aye, it was a mindless copy-paste...